inspirehep/dojson/hep/fields/bd90x99x.py

```diff
@@ -123,22 +93,40 @@ def get_value(value):
 @utils.filter_values
 def references2marc(self, key, value):
     """Produce list of references."""
+    repnos = value.get('arxiv_eprints', [])
+    if 'reportnumber' in value.get('publication_info', {}):
+        repnos.append(value['publication_info']['reportnumber'])
+    j_title = value.get('publication_info', {}).get('journal_title')
+    j_vol = value.get('publication_info', {}).get('journal_volume')
+    j_pg_s = value.get('publication_info', {}).get('page_start')
+    j_pg_e = value.get('publication_info', {}).get('page_end')
+    j_artid = value.get('publication_info', {}).get('artid')
```

Comment:
Same comment about `get_value`, and why are these `j_stuff` instead of `journal_stuff`?